### PR TITLE
fix: Fixed the copy icon when clicked on it link is copied to clipboard when a form is created

### LINKED ISSUE #104: [Bug]: Clicking on the copy icon does not copy the form link.
### Describe the bug

When the copy icon is clicked, the link does not get copied to the clipboard. As in the video I clicked on the copy icon and tried to paste it in the text area but it was not copied at all. You can see the video of it in the Screenshot section below.

### To Reproduce

1. Create a project.
2. Create a form.
3. Click on 'Copy Icon'.
4. Try to paste the link.
5. See the error that link is not copied.

If you already have a form just go to http://app.localhost:3000/dashboard/subdomain/forms and see the issue.

### Expected behavior

The expected behavior is upon clicking the copy link the link should have been copied to the clipboard.

### Screenshots/Videos

https://github.com/piyushgarg-dev/review-app/assets/131521505/a922e922-83be-4cc8-ac43-033640114f4d

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened

### PR DESCRIPTION
**Pull request by Anandateertha**
## What does this PR do?

This PR fixes the bug where the link was not copied when the user clicks on the copy button. To reproduce this follow the steps given below in the "How should this be tested?" section.

Fixes #104

BEFORE

https://github.com/piyushgarg-dev/review-app/assets/131521505/cf4e8298-019c-4127-88cc-8e79334a3136

AFTER

https://github.com/piyushgarg-dev/review-app/assets/131521505/563d95e7-bad4-4d78-ac6b-5b1e0008aefa

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create a project.
- Create a form.
- Click on 'Copy Icon'.
- Try to paste the link.
- See the error that link is not copied.

If you already have a form just go to http://app.localhost:3000/dashboard/subdomain/forms and see the issue.

### CODE REVIEW

**Comment by kartik3110** [comment]
I think the link copied should work for production as well, instead of hardcoding it with `http://localhost/${form?.slug}`
Open to any feedback and corrections.

File: components/Dashboard/FormPane.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import {
   formatToLocalDateTime,
   getTimeDistance,
 } from '@/utils/time'
+import toast from 'react-hot-toast'
```
Comment (hemantwasthere) [inline components/Dashboard/FormPane.tsx:22]: could you move this import in the above imports category

**Review by hemantwasthere — CHANGES_REQUESTED** [review]

File: components/Dashboard/FormPane.tsx (hunk 1 above)
Comment (Anandateertha) [reply components/Dashboard/FormPane.tsx]: Moved the import statement as requested Sir.

File: components/Dashboard/FormPane.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import Image from 'next/image'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
```
Comment (hemantwasthere) [inline components/Dashboard/FormPane.tsx:4]: bro add this toast import in between of these imports 😅

**Review by hemantwasthere — CHANGES_REQUESTED** [review]

**Review by hemantwasthere — APPROVED** [review]
Looks good! 👍

**Comment by Anandateertha** [comment]
> Looks good! 👍 

Thank you!

**Pull request merged**